storage/backend/nfs/mount.go

```diff
@@ -0,0 +1,72 @@
+package nfs
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/contiv/errored"
+	"github.com/contiv/volplugin/storage"
+)
+
+const (
+	mountInfoFile     = "/proc/self/mountinfo"
+	nfsMajorMountType = "0"
+)
+
+// Mounted shows any volumes that belong to volplugin on the host, in
+// their native representation. They yield a *Mount.
+func (d *Driver) Mounted(timeout time.Duration) ([]*storage.Mount, error) {
+	mounts := []*storage.Mount{}
+
+	content, err := ioutil.ReadFile(mountInfoFile)
+	if err != nil {
+		return nil, err
+	}
+
+	lines := strings.Split(string(content), "\n")
+	for _, line := range lines {
+		if len(line) == 0 {
+			continue
+		}
+
+		parts := strings.Split(line, " ")
+		devParts := strings.Split(parts[2], ":")
+		if len(devParts) != 2 {
+			return nil, errored.Errorf("Could not parse %q properly.", mountInfoFile)
+		}
+
+		if devParts[0] == nfsMajorMountType && parts[8] == "nfs4" {
```

Comment:
I don't know why this comment didn't show up here last time, but...
`cat /proc/self/mountinfo` (which I think is very clearly spelled out at the top of the function) is the file format. For editing's sake this is simpler and I don't see the value personally when you can just cat the file to display this information.